components/vf-footer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.2.1
+
+* Changes to footer based on `intranet` implementation findings [Tracking issue](https://github.com/visual-framework/vf-core/issues/1978)
```

Comment:
rename it to "EMBL-EBI Intranet"

Reply:
@sandykadam change done - https://github.com/visual-framework/vf-core/pull/2113/commits/a7b9c5286943ed09f0a25a7cfa7b6bfe6264fdba